package.json

```diff
@@ -52,7 +52,8 @@
     "textlint-rule-common-misspellings": "~1.0.1",
     "textlint-rule-no-dead-link": "~4.4.1",
     "textlint-rule-terminology": "~1.1.30",
-    "textlint-rule-write-good": "~1.6.2"
+    "textlint-rule-write-good": "~1.6.2",
+    "snyk": "^1.145.0"
```

Comment:
Similar comments to the one provided in https://github.com/telefonicaid/iotagent-ul/pull/412/files#r358410133 also apply to the packages.json file in this PR.

Reply:
This is a tool used during the analyses. I'm going to delete it.

Reply:
Fixed in 08201e79ae90920e0f9707b01629f3f281c7d4ad